webdriver/tests/get_computed_role/get.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+from six import text_type
+
+from webdriver.error import NoSuchAlertException
+
+from tests.support.asserts import assert_error, assert_success
+from tests.support.inline import inline
+
+
+def get_computed_role(session, element):
+    return session.transport.send(
+        "GET", "session/{session_id}/element/{element_id}/computedrole".format(
+            session_id=session.session_id,
+            element_id=element))
+
+
+def test_no_browsing_context(session, closed_window):
+    response = get_computed_role(session, "id")
+    assert_error(response, "no such window")
+
+
+def test_no_user_prompt(session):
+    response = get_computed_role(session, "id")
+    assert_error(response, "no such alert")
+
+
+@pytest.mark.parametrize("tag,role", [
+    ("li", "menuitem"),
+    ("input", "searchbox"),
+    ("img", "presentation")])
+def test_computed_roles(session, tag, role):
+    session.url = inline("<{0} role={1}>".format(tag, role))
```

Comment:
I think this should test some default roles as defined in https://w3c.github.io/html-aam/ as well. A naïve implementation would return the `role` attribute's value and pass this test, but wouldn't be sufficient.

For example, test:

* `<img src=x>` (expect `img`)
* `<img src=x alt>` (expect `presentation`)
* `<input type=button>` (expect `button`)
* `<input>` (expect `textbox`)

Reply:
I spoke with @AutomatedTester. He's planning on addressing this in a later PR.

Reply:
Sounds good! Have you filed an issue somewhere to track this?